### PR TITLE
STORY-2678 fetch_csv changes: removes ajax request, uses window.locat…

### PR DESCRIPTION
**Pull request by trevorfrese**
…ion change instead and adds the ext filter json values from backdraft to the url

### CODE REVIEW

**Comment by trevorfrese** [comment]
@InvokeJLa  can I get a quick CR of this?

**Comment by InvokeJLa** [comment]
I talked to Trevor in person and he's going to move some tests into this PR.

**Comment by nburwell** [comment]
:heart: :heart:

**Comment by trevorfrese** [comment]
Ah woops, also I changed the dist/backdraft.js instead of the server_side_data_table.js. next commit will have tests and proper edits of the right file.

File: spec/plugins/data_table/server_side.js [hunk 1]
```diff
@@ -894,6 +894,29 @@ describe("DataTable Plugin", function() {
         }
       });
     });
-  });
+    
+    describe("_fetchCSV", function () {
+      it("should have correct filters in parameters", function () {
```
Comment (InvokeJLa) [inline spec/plugins/data_table/server_side.js]: Nit. Just the name of this test is a missing a little (at least to me). "it should append current filter parameters to the URL when requesting a CSV export."

File: src/plugins/data_table/server_side_data_table.js [hunk 2]
```diff
@@ -137,65 +137,22 @@ var ServerSideDataTable = (function() {
...
+      else {
+        throw new Error("serverSideFiltering is expected to be enabled when _fetchCSV is called");
+      }
+    },
 
-            setTimeout(function () { URL.revokeObjectURL(downloadUrl); }, 100); // cleanup
-          }
-        }
-      })
+    _goToWindowLocation : function(url) {
```
Comment (InvokeJLa) [inline src/plugins/data_table/server_side_data_table.js]: Also, Nick, if you're watching this, we're wrapping the window.location inside a method so we can mock/spy on it in Jasmine.

File: spec/plugins/data_table/server_side.js (hunk 1 above)
Comment (trevorfrese) [inline spec/plugins/data_table/server_side.js]: addressing now

Comment (trevorfrese) [inline spec/plugins/data_table/server_side.js]: addressed

**Comment by InvokeJLa** [comment]
@trevorfrese Cool. Changes look ok to me. :+1:

**Pull request merged**